crates/swc/tests/tsc-references/classes/classDeclarations/classHeritageSpecification/classExtendsItselfIndirectly2/input.ts/es2015.1.normal/output.js

```diff
@@ -16,16 +16,16 @@ var N;
 }));
 var O;
 (function(O) {
-    class C2 extends Q1.E2 {
+    class C2 extends Q.E2 {
```

Comment:
Not a big deal because this is a usage before the declaration, so it would error anyway (maybe even now with a better error message).